### ISSUE TITLE
Unable to use v2.0.0 (go.mod need to be updated)

### ISSUE THREAD

**Issue by maeb** [samber/slog-zerolog#1]
```shell
$ go get github.com/samber/slog-zerolog@v2.0.0
go: github.com/samber/slog-zerolog@v2.0.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/samber/slog-zerolog/v2")
```

I guess the following line needs an update:
https://github.com/samber/slog-zerolog/blob/38aa5de93227d4f2f66880cc55e58de0e87f0dc5/go.mod#L1

**Comment by maeb** [comment]
Ref: https://go.dev/blog/v2-go-modules

**Comment by samber** [comment]
thanks, i made a fix -> v2.0.1
